bgpd/bgp_route.c

```diff
@@ -15113,10 +15113,10 @@ void bgp_route_init(void)
 	install_element(BGP_IPV4L_NODE, &aggregate_addressv4_cmd);
 
 	install_element(VIEW_NODE, &show_ip_bgp_instance_all_cmd);
-	install_element(VIEW_NODE, &show_ip_bgp_cmd);
 	install_element(VIEW_NODE, &show_ip_bgp_afi_safi_statistics_cmd);
 	install_element(VIEW_NODE, &show_ip_bgp_l2vpn_evpn_statistics_cmd);
-	install_element(VIEW_NODE, &show_ip_bgp_json_cmd);
+	install_element(VIEW_NODE, &show_ip_bgp_dampening_params_cmd);
+	install_element(VIEW_NODE, &show_ip_bgp_cmd);
```

Comment:
Why the whitespace changes?

Reply:
It looks like a whitespace change but it isn't :)

The last commit message says the following:
```
    * rename show_ip_bgp_cmd to show_ip_bgp_dampening_params_cmd;
    * rename show_ip_bgp_json_cmd to show_ip_bgp_cmd.
```
That was done because all options other than "dampening parameters" were moved from `show_ip_bgp_cmd` to `show_ip_bgp_dampening_params_cmd`. Then, `show_ip_bgp_json_cmd` was renamed to `show_ip_bgp_cmd` since there was no need to have `json` there anymore.